dotnet/targets/Xamarin.Shared.Sdk.DefaultItems.targets

```diff
@@ -18,6 +18,8 @@
 
     <!-- Don't include default Compile items for binding projects, because that would pick up ApiDefinition.cs and StructsAndEnums.cs -->
     <EnableDefaultCompileItems Condition=" '$(IsBindingProject)' == 'true' ">false</EnableDefaultCompileItems>
+
+    <UseMonoRuntime Condition=" '$(UseMonoRuntime)' == '' ">true</UseMonoRuntime>
```

Comment:
This is still likely going to be problematic wrt.

https://github.com/xamarin/xamarin-macios/blob/7926009d7e83512a1523b96046bce9aabf2204d1/dotnet/targets/Xamarin.Shared.Sdk.targets#L385

which is another variable that used to switch between the runtimes and applies the prefix.

Reply:
In fact, it's the same place which causes the test build failures. `_MonoNugetPackageId` is computed incorrectly which results in `_MonoFrameworkReference` and `_MonoRuntimePackPath` being incorrect leading to incorrect path for the AOT compiler eventually.

Reply:
This should fix the build failures: https://github.com/filipnavara/xamarin-macios/commit/8325f8dadcd51252c9a0935e476d8af9e55d66e7

(intentionally minimal fix; `_XamarinRuntime` can be removed later and replaced with `UseMonoRuntime` in separate PR)

Reply:
yes, some of those _dual_ variables can't be avoided (for legacy) but we should strive to remove all others

Reply:
This one is new (never released) so easy to avoid :)

Reply:
yup, better kill it before anyone gets emotionally attached to it 😄